sensor/kubernetes/listener/resources/virtualmachine/dispatcher/virtualmachines_test.go

```diff
@@ -63,7 +63,13 @@ func (s *virtualMachineSuite) Test_VirtualMachineEvents() {
 						Name:      vmName,
 						Namespace: vmNamespace,
 						Running:   false,
-					})).Times(1)
+					})).Times(1).Return(
+					&virtualmachine.Info{
+						ID:        vmUID,
+						Name:      vmName,
+						Namespace: vmNamespace,
+						Running:   false,
+					})
```

Comment:
Maybe we can add a test here that returns a CID and a running state to cover the race where the instance arrives before the VM.